src/kernel/tracelog.c

```diff
@@ -0,0 +1,568 @@
+#include <kernel.h>
+#include <http.h>
+#include <net.h>
+
+//#define TRACELOG_DEBUG
+#ifdef TRACELOG_DEBUG
+#define tracelog_debug rprintf
+#else
+#define tracelog_debug(x, ...)
+#endif
+
+#define TRACELOG_HTTP_PORT                 9091
```

Comment:
As mentioned earlier, should we just use 9090 since this is exclusive with ftrace